PED/SetPedHeadBlendData.md

```diff
@@ -8,29 +8,40 @@ ns: PED
 void SET_PED_HEAD_BLEND_DATA(Ped ped, int shapeFirstID, int shapeSecondID, int shapeThirdID, int skinFirstID, int skinSecondID, int skinThirdID, float shapeMix, float skinMix, float thirdMix, BOOL isParent);
 ```
 
-```
-The "shape" parameters control the shape of the ped's face. The "skin" parameters control the skin tone. ShapeMix and skinMix control how much the first and second IDs contribute,(typically mother and father.) ThirdMix overrides the others in favor of the third IDs. IsParent is set for "children" of the player character's grandparents during old-gen character creation. It has unknown effect otherwise.  
-The IDs start at zero and go Male Non-DLC, Female Non-DLC, Male DLC, and Female DLC.  
-!!!Can someone add working example for this???  
-try this:  
+## C# (doesn't apply to FiveM)
+```c
```

Comment:
There can only be one `c` codeblock, which should be the top one defining the native syntax. If you wish to use a codeblock in your description, leave the language definition out.

Reply:
Ah, good to know. Removed since the C# example wasn't 100% compatible with FiveM anyway. Just left a `lua` example down below.